### ISSUE TITLE
[OpenGL] Jumping Flash 1/2 - Corrupted ground textures

### ISSUE THREAD

**Issue by twinaphex** [libretro/beetle-psx-libretro#378]
Numerous garbage ground textures. Top is GL, bottom is software renderer.

https://cloud.githubusercontent.com/assets/1075274/15454961/a4f40dec-2046-11e6-8769-74756736385b.png
https://cloud.githubusercontent.com/assets/1075274/15454962/a4f84eca-2046-11e6-88e1-226413ebc15a.png

**Comment by iCatButler** [comment]
Does it still look like that top image with the latest version?

When I run the game the majority of the textures now look correct, however there appears to be an intermittent problem with the ground plane that causes it to use the wrong texture for a frame or so, causing the image to flicker.

It's easier to capture using the VRAM viewer, but that also shows that the games have a particularly strange rendering setup for a PS1 game. The output appears to be triple buffered and the ground plane is being rendered first to an offscreen target, almost like something you'd see from the Saturn's VDP2. It appears to be a means of reducing affine texture artifacts on the textures applied.

![jumping flash 2 usa -180723-103456](https://user-images.githubusercontent.com/17948959/43069745-d3d01978-8e65-11e8-8343-617831f53c1d.png)
It's this offscreen rendering that seems to have the problem of pulling in the wrong texture intermittently.
![jumping flash 2 usa -180723-103637](https://user-images.githubusercontent.com/17948959/43069807-040ce5c6-8e66-11e8-8ff5-27ea7e23ec62.png)

**Comment by twinaphex** [comment]
Yeah, it no longer looks as broken as that top picture. that is an old screenshot. Issues still persist though,

**Comment by twinaphex** [comment]
> It's this offscreen rendering that seems to have the problem of pulling in the wrong texture intermittently.

It seems the Vulkan renderer also suffers from this although far less severely.

**Comment by twinaphex** [comment]
It seems setting 'CPU Frequency Scaling Overclock' above 100% affects the flickering on the ground.

**Comment by LibretroAdmin** [comment]
All fixed now
